test/SolidityEndToEndTest.cpp

```diff
@@ -1182,12 +1182,14 @@ BOOST_AUTO_TEST_CASE(send_ether)
 	BOOST_CHECK(callContractFunction("a(address,uint256)", address, amount) == encodeArgs(1));
 	BOOST_CHECK_EQUAL(m_state.balance(address), amount);
 }
-
+// TODO: Note that these tests should actually be
```

Comment:
No-one will see this comment when that conversion is implemented. Is it really not there?
Please rather add a story to PT.